src/models/FirebaseRefProxy.ts

```diff
@@ -19,7 +19,7 @@ export const getPlanItemRef = (
   planItemId: string,
 ): RNFirebase.firestore.DocumentReference => getPlanItemsRef(studentId, planId).doc(planItemId);
 
-export const getPlanItemsRef = (studentId: string, planId: string): RNFirebase.firestore.CollectionReference =>
+export const getPlanItemsRef = (studentId: string, planId: string): any =>
```

Comment:
I had a problem with this type - I want to use `orderBy` method which is only implement in `Query` type so for now I used `any` but we definitely should fix this

Reply:
I have one idea for this :) We can keep the type `RNFirebase.firestore.CollectionReference` because it's what in fact collection returns. We'd need to update the type returned by `getChildCollectionRef` method in Plan.tsx to `RNFirebase.firestore.Query` and add this type as a part of Union type to `SubscribableModel`:
`getChildCollectionRef: () => RNFirebase.firestore.CollectionReference | RNFirebase.firestore.Query;`
It should work, cause I that `SubscribableModel` use cases we don't need anything that is specific for the CollectionReference and not exist in a Query type :) checked it myself and it looks fine :)